docs/niswitch/functions.rst

```diff
@@ -133,7 +133,7 @@ niswitch.Session methods
 
     :type scan_mode: :py:data:`niswitch.ScanMode`
 
-.. py:method:: configure_scan_trigger(trigger_input, scan_advanced_output, scan_delay='datetime.timedelta(seconds=0.0)')
+.. py:method:: configure_scan_trigger(trigger_input, scan_advanced_output, scan_delay=datetime.timedelta(seconds=0.0))
```

Comment:
If you could factor out the change that fixed the stray quote problem out to its own PR that would be great for reviewers (aka me).

Reply:
You want more PRs to review? I can do that.

Reply:
I always prefer several small, noisy, yet uncontroversial PRs than single large and difficult to diff PR that will require several rounds.

I appreciate it!